stations.csv

```diff
@@ -8934,7 +8934,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 10809;Braunschweig Hbf;braunschweig-hbf;8013240;;52.252236;10.539699;;f;DE;f;Europe/Berlin;t;;;f;;f;8000049;t;;f;;f;;f;;;;f;;f;;f;;f;DEBSW;f;f;;;Main station;Brunswick Estación central;Gare centrale;Stazione centrale;;;;;ブラウンシュヴァイク;브라운슈바이크;Brunszwik;;Брауншвейг;;;不伦瑞克
 10810;Bünde (Westf);bunde-westf;8021173;;52.202076;8.573875;;f;DE;f;Europe/Berlin;t;DEAFG;;f;;f;8000059;t;;f;;f;;f;;;;f;;f;;f;;f;DEAFG;f;f;;;;;;;;;;;;;;;Бюнде;;;
 10811;Siershahn;siershahn;8019581;;50.485388;7.771561;;f;DE;f;Europe/Berlin;t;;;f;;f;8000060;t;;f;;f;;f;;;;f;;f;;f;;f;DESHN;f;f;;;;;;;;;;;;;;;;;;
-10812;Burg-u. Nieder Gemünden;burg-u-nieder-gemunden;8005610;;50.691115;9.047606;;f;DE;f;Europe/Berlin;t;;;f;;f;8000061;t;;f;;f;;f;;;;f;;f;;f;;f;DEBGE;f;f;;;;;;;;;;;;;;;;;;
+10812;Burg- und Nieder-Gemünden;burg-und-nieder-gemunden;8005610;;50.691115;9.047606;;f;DE;f;Europe/Berlin;t;;;f;;f;8000061;t;;f;;f;;f;;;;f;;f;;f;;f;DEBGE;f;f;;;;;;;;;;;;;;;;;;
```

Comment:
Yes, it's intended.
https://de.wikipedia.org/wiki/Bahnstrecke_Kirchhain%E2%80%93Burg-_und_Nieder-Gem%C3%BCnden